# origin: bk (joshk)
# cset: 1.1447.4.11
# inclusion: n/a (projected: 2.4.28-pre3)
# description: add missing MODULE_LICENSE("GPL") to many files
# revision date: 2004-09-03

--- kernel-source-2.4.26/drivers/fc4/fc.c~	2004-08-02 10:09:54.000000000 -0700
+++ kernel-source-2.4.26/drivers/fc4/fc.c	2004-08-02 10:10:09.000000000 -0700
@@ -1155,3 +1155,5 @@
 	kfree(p);
 	return status;
 }
+
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/sbus/char/bpp.c~	2004-08-02 10:08:31.000000000 -0700
+++ kernel-source-2.4.26/drivers/sbus/char/bpp.c	2004-08-02 10:10:33.000000000 -0700
@@ -1073,3 +1073,5 @@
 
 module_init(bpp_init);
 module_exit(bpp_cleanup);
+
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/sbus/char/vfc_dev.c~	2004-08-02 10:08:39.000000000 -0700
+++ kernel-source-2.4.26/drivers/sbus/char/vfc_dev.c	2004-08-02 10:10:48.000000000 -0700
@@ -743,4 +743,4 @@
 }
 #endif
 
-
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/scsi/esp.c~	2004-08-02 10:04:30.000000000 -0700
+++ kernel-source-2.4.26/drivers/scsi/esp.c	2004-08-02 10:06:45.000000000 -0700
@@ -4368,4 +4368,4 @@
 
 #include "scsi_module.c"
 
-EXPORT_NO_SYMBOLS;
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/scsi/fcal.c~	2004-08-02 10:08:51.000000000 -0700
+++ kernel-source-2.4.26/drivers/scsi/fcal.c	2004-08-02 10:11:13.000000000 -0700
@@ -300,4 +300,4 @@
 
 #include "scsi_module.c"
 
-EXPORT_NO_SYMBOLS;
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/scsi/pluto.c~	2004-08-02 10:09:25.000000000 -0700
+++ kernel-source-2.4.26/drivers/scsi/pluto.c	2004-08-02 10:11:33.000000000 -0700
@@ -339,4 +339,4 @@
 
 #include "scsi_module.c"
 
-EXPORT_NO_SYMBOLS;
+MODULE_LICENSE("GPL");
--- kernel-source-2.4.26/drivers/scsi/qlogicpti.c~	2004-08-02 10:09:30.000000000 -0700
+++ kernel-source-2.4.26/drivers/scsi/qlogicpti.c	2004-08-02 10:11:53.000000000 -0700
@@ -1533,4 +1533,4 @@
 
 #include "scsi_module.c"
 
-EXPORT_NO_SYMBOLS;
+MODULE_LICENSE("GPL");
